website/content/sdk/guides/advanced/api-keys.md

```diff
@@ -1,6 +1,8 @@
 ---
 title: Set API Keys
 sidebar_position: 1
+description: With the OpenBB SDK you can define your, often free, API key from various platforms like Alpha Vantage, FRED, IEX, Twitter, DeGiro, Binance and Coinglass. An API key is a set of random characters provided to you by a third party vendor that allows you to access data via their API endpoints. API keys can be managed directly from the SDK environment.
+keywords: [api, keys, api keys, data provider, data, free, alpha vantage, fred, iex, twitter, degiro, binance, coinglass, polygon, intrinio, openbb sdk]
```

Comment:
iirc it should be 'api, keys, .....' and not []

Reply:
If I do that it gives me errors that it isn't an array.

<img width="824" alt="image" src="https://user-images.githubusercontent.com/46355364/214862407-e60765e7-ca61-4bac-bfff-868e1d89348f.png">

Reply:
Docs say it too: https://docusaurus.io/docs/next/seo

Reply:
oooh aight! i didnt think docusaurus was abstracting that. (that's the way it works on html)